inductiva/stellarators/scenarios/coils/coils.py

```diff
@@ -26,6 +28,48 @@ def __init__(self, coils, num_field_periods):
         self.num_coils = len(coils)
         self.num_field_periods = num_field_periods
 
+    @classmethod
+    def from_circular_curves(cls, num_field_periods, num_coils, coil_currents,
+                             major_radius, minor_radius):
+        """Create simple circular and equally spaced curves.
```

Comment:
You could explain here briefly which coefficients of the Fourier series are non-zero.